frontend/src/features/projects/spl/forms/SurplusPropertyListForm.tsx

```diff
@@ -198,7 +206,6 @@ const SurplusPropertyListForm = ({
           <TooltipIcon toolTipId="dateEnteredMarket" toolTip={dateEnteredMarket} />
         </Form.Label>
         <FastDatePicker
-          required
```

Comment:
Where these listed as required inappropriately before? This change seems to be about adding some new fields so changing the required status on some of these other date fields seems unrelated.

Reply:
They're only required under certain conditions.  Doesn't really make sense to label them as always required.